Add reusable base release drafter config

mkurz opened the pull request.

mkurz commented:
Let's see if this works, so we avoid all those duplicate files in each repo.

ihostage commented:
© [Release Drafter Docs](https://github.com/marketplace/actions/release-drafter#configuration-options)
> Release Drafter also supports Probot Config, if you want to store your configuration files in a central repository. This allows you to share configurations between projects, and create a organization-wide configuration file by creating a repository named .github with the file .github/release-drafter.yml 

Will you try to use Probot for extending configuration? Do I right understand?

ihostage commented:
Ah, I already see https://github.com/playframework/twirl/pull/488

mkurz commented:
Turns out I spend too much time to get this working :innocent:, in particular because I stumbled into bugs in release-drafter and even in probot. I couldn't help myself and and went ahead to fix those bugs:
* https://github.com/release-drafter/release-drafter/issues/1048 fixed by https://github.com/release-drafter/release-drafter/pull/1049
* Added a header and footer feature, so we can reuse the premium sponsor footer across repos: https://github.com/release-drafter/release-drafter/issues/1047 fixed by https://github.com/release-drafter/release-drafter/pull/1050
* Fixed a bug regarding the `_extends` feature in probot which prevents us from extending a release-draft from another one if they are located in the `.github` repo: https://github.com/probot/octokit-plugin-config/issues/166 fixing here: https://github.com/probot/octokit-plugin-config/pull/167

I hope it was worth it, I need to work on more meaningful things again :wink:
